README.md

```diff
@@ -30,27 +30,48 @@ variable `BIGML_SENSENET_CACHE_PATH`.
 
 ## Model Instantiation
 
-To instantiate a model, pass the model specification and the dict
-of additional, optional settings to `create_model`.  For example:
+To instantiate a model, pass the model specification and the dict of
+additional, optional settings to `models.wrappers.create_model`.  For
+example:
 
 ```
-model = create_model(a_dict, settings={'image_path_prefix': 'images/path/'})
+model = wrappers.create_model(a_dict, settings={'image_path_prefix': 'images/path/'})
 ```
 
 Again, `a_dict` is typically a downloaded BigML model, read into a
-python dictionary via `json.load` or similar.
+python dictionary via `json.load` or similar.  You may also pass the
+path to a file containing such a model:
+
+```
+model = wrappers.create_model('model.json', settings=None)
+```
 
-You may also pass the path to a file containing such a model:
+A similar function, `models.wrappers.create_image_feature_extractor`,
+allows clients to create a model object that returns instead the
+outputs of the final global pooling or flattening layer of the image
+model, given an image as input:
 
 ```
-model = create_model('model.json', settings=None)
+extractor = create_image_feature_extractor("resnet18", None)
+extractor("path/to/image.jpeg").shape # (1, 512)
 ```
 
-For image models, `settings` is either `None` (the default) or a dict
-of optional settings which may contain:
+Note that this only works for networks with at least one image input,
+and does not work for bounding box models, as there is no global
+pooling or flattening step in those models.
+
+For both `craete_image_feature_extractor` and `create model`,
```

Comment:
> I guess bounding_box_threshold and iou_threshold can only be used in create_model. Any other restriction?

I actually don't restrict anywhere the settings that you can pass in (other than to limit them to the keys we know about); you can pass `bounding_box_threshold` everywhere, and it will just be ignored in cases where it isn't relevant.

Reply:
> That said, what about history of gradients and other features not directly related to deepnet layers?

That is not here.  Those are currently only implemented in Java, and will have to be reimplemented in python (probably numpy).